XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs

```diff
@@ -107,8 +109,56 @@ private void InitializeData()
 
             SampleCodeLayout2.Value = @"<common:ActivityFeedLayout x:Key=""MyFeedLayout"" ColumnSpacing=""12""
                           RowSpacing=""12"" MinItemSize=""80, 108""/>";
+
+
+            // Initialize list of colors for animated scrolling sample
+            IList<String> colors = (typeof(Colors).GetRuntimeProperties().Select(c => c.ToString())).ToList();
+            for (int i =0; i < colors.Count(); i++)
+            {
+                colors[i] = colors[i].Substring(17);
+
+            }
+            
+            animatedScrollRepeater.ItemsSource = colors;
```

Comment:
>colors [](start = 49, length = 6)

creating the list of colors can be extracted into its own method to avoid seeing its details here..
so something like
animatedScrollRepeater.ItemsSource = GetListOfColors();

Reply:
Cleaned it up, but put the recipe generator method outside the Recipe class as it needed to use the colors as well